RecoLocalMuon/CSCSegment/src/CSCSegAlgoRU.cc

```diff
@@ -391,11 +414,10 @@ bool CSCSegAlgoRU::areHitsCloseInR(const AlgoState& aState, const CSCRecHit2D* h
     h1z = 1;
     h2z = 1;
   }
-  if (gp2.perp() > ((gp1.perp() - aState.dRMax*aState.strip_iadd*maxWG_width[iStn])*h2z)/h1z && gp2.perp() < ((gp1.perp() + aState.dRMax*aState.strip_iadd*maxWG_width[iStn])*h2z)/h1z){
-    return true;
-  }
-  else{
-    return false;
+  if(enlarge) {
+    return (gp2.perp() > ((gp1.perp() - aState.dRMax*aState.strip_iadd*maxWG_width[iStn])*h2z)/h1z && gp2.perp() < ((gp1.perp() + aState.dRMax*aState.strip_iadd*maxWG_width[iStn])*h2z)/h1z)? true:false;
+  }else{
+    return (gp2.perp() > ((gp1.perp() - aState.dRMax*maxWG_width[iStn])*h2z)/h1z && gp2.perp() < ((gp1.perp() + aState.dRMax*maxWG_width[iStn])*h2z)/h1z)? true:false;
```

Comment:
please add some newlines to improve readability

Reply:
> please add some newlines to improve readability

sorry for being unclear, I meant to ask for line breaks in these long lines of code. Simple addition of empty lines around them is not really improving the code readability.